lib/src/calendar.dart

```diff
@@ -248,7 +248,13 @@ class _TableCalendarState extends State<TableCalendar> with SingleTickerProvider
 
   void _selectedDayCallback(DateTime day) {
     if (widget.onDaySelected != null) {
-      widget.onDaySelected(day, widget.calendarController.visibleEvents[_getEventKey(day)] ?? []);
+      widget.onDaySelected(
```

Comment:
I would prefer to keep list of holidays separate from list of events (similarly to [markersBuilder](https://pub.dev/documentation/table_calendar/latest/table_calendar/CalendarBuilders/markersBuilder.html)):
```dart
onDaySelected(day, events, holidays)
```

Reply:
I think its sorted as requested...

However. this is a breaking change for people who are using it

Reply:
That's correct. I do plan on releasing a `3.0.0` version with some breaking changes, so it will be added there as well :)